tests/conftest.py

```diff
@@ -169,7 +169,7 @@ def md_content_config():
     return markdown_config
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture(scope="session")
```

Comment:
Do you know which commit this change is from/why it's required?

Reply:
No, it maybe a borked merge. In my minimal testing this didn't affect tests, but I can revert this change if required